packages/core/README.md

```diff
@@ -66,12 +67,16 @@ Use the [Vue bindings](../vue/README.md).
 - Edit `src/main.js` to include:
 
   ```js
-  // Import Way2Web Web Components
+  // Import Way2Web Design System components
   import { defineCustomElements } from '@w2wds/core/loader';
+
+  /* Core CSS required for Way2Web Design System components to work properly */
+  import '@w2wds/core/dist/core/core.css';
```

Comment:
I have to validate this when this PR is merged and I've done a publish to NPM. Doesn't work right now.